index.js

```diff
@@ -1,13 +1,43 @@
-const mongoose = require('mongoose');
-const Recipe = require('./models/Recipe.model'); // Import of the model Recipe from './models/Recipe.model.js'
-const data = require('./data.js'); // Import of the data from './data.js'
+const mongoose = require("mongoose");
+const Recipe = require("./models/Recipe.model"); // Import of the model Recipe from './models/Recipe.model.js'
+const data = require("./data.js"); // Import of the data from './data.js'
 
 // Connection to the database "recipeApp"
 mongoose
-  .connect('mongodb://localhost/recipe-app-dev', {
+  .connect("mongodb://localhost/recipe-app-dev", {
     useCreateIndex: true,
     useNewUrlParser: true,
     useUnifiedTopology: true
   })
-  .then(x => console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`))
-  .catch(err => console.error('Error connecting to mongo', err));
+  .then(x =>
+    console.log(`Connected to Mongo! Database name: "${x.connections[0].name}"`)
+  )
+  .catch(err => console.error("Error connecting to mongo", err));
+
+Recipe.create({
+  title: "Pizza",
+  level: "UltraPro Chef",
+  ingredients: ["tomatos", "cheese", "olives"],
+  cuisine: "italian",
+  dishType: "Dish",
+  duration: 10,
+  creator: "Markus"
+}).then(created => {
+  console.log(created);
+});
+
+Recipe.insertMany(data).then(inserted => {
+  console.log(inserted);
+});
+
+Recipe.updateOne({ title: "Rigatoni alla Genovese" }, { duration: 100 }).then(
+  result => {
+    console.log(result);
+  }
+);
+
+Recipe.deleteOne({ title: "Carrot Cake" }).then(deleted => {
+  console.log(deleted);
+});
```

Comment:
Also I encourage you to use the catch method as well to be able to see the erros if your database crashes.
It will save you a lot of trouble in the future :)
Errors are our friends